appinfo/routes.php

```diff
@@ -245,6 +245,15 @@
 				'token' => '^[a-z0-9]{4,30}$',
 			],
 		],
+		[
+			'name' => 'Room#setListable',
+			'url' => '/api/{apiVersion}/room/{token}/listable',
+			'verb' => 'PUT',
+			'requirements' => [
+				'apiVersion' => 'v(1|2|3)',
```

Comment:
should this be limited to v3 ? @nickvergessen

Reply:
yes